File: tests/parser_integration/fixtures/python/string_literals.py
# Python string literals for testing

MESSAGE = "Hello, world!"
GREETING = "Welcome to the application"
ERROR_MSG = "An error occurred while processing"

def show_message():
    print(MESSAGE)

def handle_error():
    print("Error: operation failed")

class Example:
    def __init__(self):
        self.name = "Example class"
        
    def greet(self, name):
        return f"Hello, {name}!"

File: tests/parser_integration/fixtures/rust/string_literals.rs
// Rust string literals for testing
fn main() {
    let message = "Hello, world!";
    let greeting = "Welcome to the application";
    let error_msg = "An error occurred while processing";
    
    println!("{}", message);
}

fn show_message() {
    println!("Displaying message");
}

fn handle_error() {
    println!("Error: operation failed");
}

fn format_string(name: &str) -> String {
    format!("Hello, {}!", name)
}

